### PR TITLE
Help Page Enhancements

### PR DESCRIPTION
**Pull request by esterlinej**
- add breadcrums
- google no translate

### CODE REVIEW

**Comment by esterlinej** [comment]
Screen Shot 2020-10-05 at 11 27 12 AM

**Comment by mikeykhalil** [comment]
Do we currently make use of the `lang` attribute? This should inform browsers about the language used in the page. 
https://www.w3schools.com/tags/ref_language_codes.asp

**Comment by esterlinej** [comment]
> Do we currently make use of the `lang` attribute? This should inform browsers about the language used in the page.
> https://www.w3schools.com/tags/ref_language_codes.asp

Will test this out in DQAs, if works will utilize this instead.

**Comment by esterlinej** [comment]
Screen Shot 2020-10-05 at 11 53 32 AM

File: ui/views/help.html [hunk 1]
```diff
@@ -1,43 +1,62 @@
-<html>
+<html lang="en">
     <head>
         {{ template "header-head" . }}
         <script type="text/javascript">
             const apiBaseURL = "{{ .APIBaseURL }}";
         </script>
+        <style>
```
Comment (bkeroackdsc) [inline ui/views/help.html:7]: I'm fine with this but if it gets any more complex (this goes for other views too), we should split out to a separate CSS file (and then see if we can de-dupe into a set of shared styles in a partial).

**Review by bkeroackdsc — APPROVED** [review]

**Review by mikeykhalil — APPROVED** [review]

**Pull request merged**